### PR TITLE
Fix CI badge link

### PR DESCRIPTION
**Pull request by osamingo**
SSIA

### CODE REVIEW

**Comment by osamingo** [comment]
@dylan-bourque If you have time, could you review my PR?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # UUID
 
 [![License](https://img.shields.io/github/license/gofrs/uuid.svg)](https://github.com/gofrs/uuid/blob/master/LICENSE)
-[![Build Status](https://travis-ci.org/gofrs/uuid.svg?branch=master)](https://travis-ci.org/gofrs/uuid)
+[![Go](https://github.com/gofrs/uuid/actions/workflows/go.yml/badge.svg)](https://github.com/gofrs/uuid/actions/workflows/go.yml)
```
Comment (dylan-bourque) [inline README.md:4]: The link update is 👍 but I would probably leave the label set to "Build Status"

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 [![GoDoc](http://godoc.org/github.com/gofrs/uuid?status.svg)](http://godoc.org/github.com/gofrs/uuid)
```
Comment (dylan-bourque) [inline README.md:5]: As long as you're cleaning up this README, would you mind updating this link to point to pkg.go.dev?

**Review by dylan-bourque — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (osamingo) [reply README.md:4]: @dylan-bourque This label depends on the name of the GitHub Actions, so if we want it to be `Build`, we'll need to change `name` field in `go.yml`.

https://github.com/gofrs/uuid/blob/master/.github/workflows/go.yml#L1

Comment (dylan-bourque) [reply README.md:4]: The alt text for the image in the Markdown has to match the name of the workflow?  That doesn't make sense and it's not called out that way in the docs anywhere that I can find.

Comment (osamingo) [reply README.md:4]: The `alt text` one, that's ok, I mistook it for the label shown on the badge.

Comment (osamingo) [reply README.md]: I fixed it 👍🏼

**Review by dylan-bourque — APPROVED** [review]
LGTM

**Comment by cameracker** [comment]
Thank you very much for catching this mistake and fixing it @osamingo !

**Pull request merged**